src/transformers/models/poolformer/image_processing_poolformer.py

```diff
@@ -184,10 +184,7 @@ def resize(
                 if size["height"] == size["width"]:
                     scale_size = int(math.floor(size["height"] / crop_pct))
                 else:
-                    scale_size = (
-                        int(math.floor(size["height"] / crop_pct)),
-                        int(math.floor(size["width"] / crop_pct)),
-                    )
+                    scale_size = (int(size["height"] / crop_pct), int(size["width"] / crop_pct))
```

Comment:
I am wondering if this change is necessary to have the test pass. I see this is what has been in the previous version, but don't think this matter.

In fact, `math.floor` in this file is not necessary (at least for Python 3.x). Anyway, using it or not are both fine to me, but keeping the same style would be good (as long as my claim is correct - i.e. it doesn't affect the test we are trying to fix)

Reply:
You're completely right. Tbh, I was just porting from the old feature extractors without thinking too much. I'll remove the `math.floor` in the function to make it tidier 🧹